java/server/src/org/openqa/selenium/grid/data/NodeStatus.java

```diff
@@ -110,12 +117,13 @@ public int hashCode() {
   }
 
   private Map<String, Object> toJson() {
-    return ImmutableMap.of(
```

Comment:
I had it as a Map but IntelliJ really wants to switch it to a Map. Otherwise it doesn't compile. I feel like there is an obvious java-ism i am missing.